src/constants/pools.ts

```diff
@@ -112,7 +116,9 @@ const POOLS_MAINNET: Pools = {
       '0x5c6ee304399dbdb9c8ef030ab642b10820db8f56000200000000000000000014'
   },
   Pagination: {
-    PerPage: 10
+    PerPage: 10,
+    PerPool: 10,
```

Comment:
Why do we need both of these?

Reply:
in order not to mix logics of pool transaction tables and other ones